packages/manager/apps/telecom/src/app/telecom/pack/migration/confirm/pack-migration-confirm.html

```diff
@@ -4,266 +4,288 @@
         <h3 data-translate="telecom_pack_migration_doing"></h3>
     </div>
 
-    <div class="widget-presentation" data-ng-if="!$ctrl.loading.migrate">
-        <h2
-            class="widget-presentation-title"
-            data-translate="telecom_pack_migration_confirm_resume_title"
-        ></h2>
+    <form name="resumeForm" data-ng-submit="$ctrl.startMigration()">
+        <div class="widget-presentation" data-ng-if="!$ctrl.loading.migrate">
+            <h2
+                class="widget-presentation-title"
+                data-translate="telecom_pack_migration_confirm_resume_title"
+            ></h2>
 
-        <p
-            data-translate="telecom_pack_migration_confirm_migrate_to"
-            data-translate-values="{ fromOffer : $ctrl.process.pack.offerDescription, toOffer : $ctrl.process.selectedOffer.description }"
-        ></p>
+            <p
+                data-translate="telecom_pack_migration_confirm_migrate_to"
+                data-translate-values="{ fromOffer : $ctrl.process.pack.offerDescription, toOffer : $ctrl.process.selectedOffer.description }"
+            ></p>
+
+            <div>
+                <div class="responsive-table-line my-3">
```

Comment:
Not using a oui-datagrid ?

Reply:
Nope. Like we already discussed on this subject a time ago, an oui-datagrid componant could not be applied here.